test/cpp/api/modules.cpp

```diff
@@ -2955,7 +2955,7 @@ TEST_F(ModulesTest, GELU) {
   const auto x = torch::linspace(-3.0, 3.0, 100);
   const auto y_exp = x * 0.5 * (1.0 + torch::erf(x / std::sqrt(2.0)));
   const auto y = model(x);
-  ASSERT_TRUE(torch::allclose(y, y_exp));
+  ASSERT_TRUE(torch::allclose(y, y_exp, 1e-3, 0));
```

Comment:
Why is this different from https://github.com/pytorch/pytorch/pull/58525/files?diff=unified&w=1#diff-9963e49433a377372e1a24ddb77b72dc1e3e0f4e44e49a24d7dfed55c85177daR1000 ? Also, in the PR summary, can we explicitly say what the precision difference between MKLDNN's implementation and the original implementation is?

Reply:
Here should be `ASSERT_TRUE(torch::allclose(y, y_exp, 1.3e-06, 1e-05));`, which precision is same as that test case in functional.cpp. 
The explanation of the precision difference is https://github.com/oneapi-src/oneDNN/issues/1122#issuecomment-884621774.